core/storage/user/gae_models.py

```diff
@@ -927,13 +927,20 @@ def export_data(user_id):
         if user_model is None:
             return {}
 
+        creator_user_models = UserSettingsModel.get_multi(
+            user_model.creator_ids)
```

Comment:
Maybe use (as it is more readable):
```suggestion
        creator_user_models = (
            UserSettingsModel.get_multi(user_model.creator_ids))
```